examples/heatTransfer/write/IO_hdf5_a.cpp

```diff
@@ -49,7 +49,7 @@ void IO::write(int step, const HeatTransfer &ht, const Settings &s, MPI_Comm com
     double total_time = MPI_Wtime() - time_start;
     uint64_t adios_totalsize = 2 * sizeof(int) + 2 * s.ndx * s.ndy * sizeof(double);
     uint64_t sizeMB = adios_totalsize * s.nproc / 1024 / 1024 / 1024; // size in MB
-    uint64_t mbs = sizeMB / total_time;
+    double mbs = sizeMB / total_time;
```

Comment:
```suggestion
    double mbs = sizeMB / static_cast<double>(total_time);
```

C/C++ casting rules, I made an example here https://godbolt.org/z/rvWGzPKoj

Reply:
I could be misinterpreting your suggestion, but isn't `total_time` already of type `double`?  Though you're correct about division when both operands are of type `int`, I think as long as one operand is of type `double`, the other will be promoted before the operation.  Let me know if I misunderstood though.